zerver/tests/test_events.py

```diff
@@ -1471,9 +1471,16 @@ def test_default_streams_events(self) -> None:
             ('type', equals('default_streams')),
             ('default_streams', check_list(check_dict([
```

Comment:
Let's change this to `check_dict_only` if we can.

Reply:
Could also extract `check_stream = check_dict[_only](…)` and reuse it in the next hunk.

Reply:
I did a deeper dive on the stream fields that addresses both concerns here.